cli/bench/http/deno_reactdom_ssr_flash.jsx

```diff
@@ -18,12 +18,6 @@ const headers = {
   },
 };
 
-serve(
-  {
-    fetch: async () => {
-      return new Response(await renderToReadableStream(<App />), headers);
-    },
-    hostname,
-    port,
-  },
-);
+serve({ hostname, port }, async () => {
```

Comment:
Handler should be the first arg

Reply:
Why? It's easier to read if options come before the handler function (more glancable).

Reply:
Okay I see you made 3 different overloads...